openbb_terminal/stocks/fundamental_analysis/av_model.py

```diff
@@ -19,6 +19,19 @@
 logger = logging.getLogger(__name__)
 
 
+def check_premium_key(json_response: Dict) -> bool:
+    """Checks if the response is the premium endpoint"""
+    if json_response == {
+        "Information": "Thank you for using Alpha Vantage! This is a premium endpoint. You may subscribe to "
+        "any of the premium plans at https://www.alphavantage.co/premium/ to instantly unlock all premium endpoints"
+    }:
+        console.print(
```

Comment:
There is probably a better way than than printing message in the model.

Reply:
There probably is but this is the way we have been doing them throughout.  @colin99d wants a consistent way